src/components/Preferences/ModalActions.vue

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+import Vue from 'vue';
+export default Vue.extend({ name: 'preferences-actions' });
+</script>
+
+<template>
+  <div class="preferences-actions">
+    <button class="btn role-primary">
+      Apply
+    </button>
+    <button class="btn role-secondary">
+      Cancel
+    </button>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+  .preferences-actions {
+    display: flex;
+    flex-direction: row-reverse;
```

Comment:
Should that be mac-only? (IIRC, mac + Gnome puts the apply/OK button in the end corner, and Windows + KDE always uses [OK, cancel] ordering.)

Reply:
Perhaps I'm abusing `flex-direction` to ensure that items are rendered at the end of the flex container and `justify/align-content` would be a more appropriate for this use case. I'll explore in a follow-up PR.